ai-platform-unified/notebooks/unofficial/hyperparameter_tuning/distributed-hyperparameter-tuning.ipynb

```diff
@@ -0,0 +1,879 @@
+{
```

Comment:
<p>missing the argument for search algorithm (set it to the default None)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Done